agent/consul/acl_endpoint_legacy.go

```diff
@@ -169,7 +169,7 @@ func (a *ACL) Apply(args *structs.ACLRequest, reply *string) error {
 	// NOTE: We will not support enterprise authorizer contexts with legacy ACLs
 	if rule, err := a.srv.ResolveToken(args.Token); err != nil {
```

Comment:
minor: What if rule here (and other places calling `ResolveToken*`) was renamed to authz and the variable name rule was reserved for binding rules?

Reply:
I think that's a great idea, I'll do that as a follow up